JavaScript/React.md

```diff
@@ -210,7 +210,7 @@ class ComponentName extends React.Component {
 }
 ```
 
-Avoid writing any logic inline. Extract it to the outside of the JSX block.
+Avoid writing any logic in line. Extract it to the outside of the JSX block.
```

Comment:
Think it should stay "inline"

Reply:
I don't think inline can be used as an adverb. If it should stay inline, and inline is an adjective, would it make sense to change places and write; Avoid writing any inline logic?

Reply:
I would change it to that 👍